test.js

```diff
@@ -1,36 +1,59 @@
 'use strict';
-var assert = require('assert');
+var assert = require('chai').assert;
```

Comment:
> I'm for disabling the offending test, and filing an issue to follow back around with the issue later.

:+1: 

-

@lzilioli Can you also update the readme to mention this limitation?